spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -176,7 +176,8 @@ object SpdxLicenseAliasMapping {
     /**
      * The map of varied SPDX license ids associated with their corresponding SPDX expression.
      */
-    val mapping = customLicenseIds + deprecatedLicenseIds + deprecatedExceptionIds
+    val mapping =
+        (customLicenseIds + deprecatedLicenseIds + deprecatedExceptionIds).toSortedMap(String.CASE_INSENSITIVE_ORDER)
```

Comment:
Good finding. However, I think I'd prefer to call `.toSortedMap(String.CASE_INSENSITIVE_ORDER)` when declaring `deprecatedLicenseIds` and `deprecatedExceptionIds`, similar to like we do when declaring `customLicenseIds`. Or would the `+` operator here create a new case-sensitive map here then again?

Reply:
Yes, it is required, the `+` operator creates a `LinkedHashMap`.

Reply:
Just one more comment on the kind of line break here: Personally, I prefer to break the line at the *last* occurrence of a non-word character, i.e.

    val mapping = (customLicenseIds + deprecatedLicenseIds + deprecatedExceptionIds)
        .toSortedMap(String.CASE_INSENSITIVE_ORDER)

This formatting also is slightly more convenient IMO when e.g. do a global search for "val mapping" as then the single-line summaries would show something meaningful.